docs/photoniq/event-delivery/get-started-event-delivery.md

```diff
@@ -9,42 +9,40 @@ However, some people like to be more hands-on with their management. This page p
 
 ## Prerequisites
 
-- If you want to work with Macrometa streams, then you must have a [Macrometa account](https://auth-play.macrometa.io/) with sufficient permissions to create collections. Your Macrometa partners can help you get set up.
-- An application to receive Event Delivery streams.
-
-## Set Up Macrometa Streams
-
-Event Delivery lets you filter events from Macrometa streams and publish events to streams, so the first step is to get your data into a stream in the Macrometa Global Data Network (GDN).
+- To work with Macrometa streams, then you must have a [Macrometa account](https://auth-play.macrometa.io/) with sufficient permissions to create collections. Your Macrometa partners can help you get set up, or you can sign up on your own.
+- Decide what kind of [collection](../../collections/) to send streaming data to. Consult with your Macrometa partners to decide what is the best option for your use case.
+  - [Document Collection](../../collections/documents/) - Accepts any document type.
+  - [Key-Value Collection](../../collections/keyvalue/) - Accepts key-value pairs, can be set up to include blobs.
 
-You have several options for streams:
-
-- Local stream
-- Global stream
-- Collection stream
+- An application to receive Event Delivery streams.
+- Plan your filters, which are set by adding queries to event subscriptions. For more information, refer to [Event Delivery Queries](event-delivery-queries.md).
 
-Consult with your Macrometa partners to decide what is the best option for your use case. More information on these options is below, and they are fully documented in the [Macrometa GDN documentation](../../index.md).
+## Set Up Your Stream in the GDN
 
-### Local or Global Stream
+Event Delivery lets you filter events from Macrometa streams and publish events to streams, so the first step is to get your data into a collection stream in the Macrometa Global Data Network (GDN).
 
 A stream is a named channel for sending messages. Each stream is backed by a distributed append-only log and can be local (at one edge location only) or global (across all edge locations in the [fabric](../../geofabrics/)).
 
-For more information, refer to [Streams](../../streams/).
-
-### Collection Stream
+1. (Optional) [Create a fabric](../../geofabrics/create-geofabric.md) for your collection. This allows you to decide exactly where your data lives. If you skip this step, then the collection will probably be in the system fabric.
```

Comment:
I think we should remove the probably word from this sentence.

Reply:
Removed, it just seems inaccurate. I mean, what if someone else on the account made a fabric and switched to it?